### DISCUSSION TITLE
Can org files support been added?

### DISCUSSION THREAD

**Discussion by basheewang** [tefkah/emacs-docs#8] [Ideas]
It looks more often to use org than md files for emacers.  Thank you!

**Comment by tefkah** [comment]
I don't actually write the MD files myself, I just generate them from the main documentation html! 

The documentation framework im using only supports md files so this is kind of a hard ask. Theoretically the md files could be generated from org files, but since only (afaik) the org documentation is actually written in org it seems like an unecessary extra step
